### PR TITLE
feat: Add image parameter to create_scheduled_event

### PR DESCRIPTION
**Pull request by Snawe**
## Summary

The parameter `cover` was missing in the `create_scheduled_event()` function

## Information

- [x] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.

### CODE REVIEW

**Comment by Snawe** [comment]
during fixing this issue, I saw that the payload parameter is actually called `image` and not cover... Would it make sense to rename `cover` at the create and edit command to `image`? (But that would be a breaking change!)

**Comment by Dorukyum** [comment]
> during fixing this issue, I saw that the payload parameter is actually called `image` and not cover... Would it make sense to rename `cover` at the create and edit command to `image`? (But that would be a breaking change!)

The API wrapper should reflect the API, meaning the parameter should be called `image`.

**Comment by Snawe** [comment]
> > during fixing this issue, I saw that the payload parameter is actually called `image` and not cover... Would it make sense to rename `cover` at the create and edit command to `image`? (But that would be a breaking change!)
> 
> The API wrapper should reflect the API, meaning the parameter should be called `image`.

Ok. Will change that as well. Should I attach the commit to this PR? Or should I create a new PR since it is a breaking change?

File: discord/guild.py [hunk 1]
```diff
@@ -3650,6 +3650,7 @@ async def create_scheduled_event(
         location: str | int | VoiceChannel | StageChannel | ScheduledEventLocation,
         privacy_level: ScheduledEventPrivacyLevel = ScheduledEventPrivacyLevel.guild_only,
         reason: str | None = None,
+        cover: bytes = MISSING,
```
Comment (Lulalaby) [inline discord/guild.py:3653]:
```suggestion
        image: bytes = MISSING,
```

File: discord/guild.py [hunk 2]
```diff
@@ -3672,6 +3673,8 @@ async def create_scheduled_event(
             so there is no need to change this parameter.
         reason: Optional[:class:`str`]
             The reason to show in the audit log.
+        cover: Optional[:class:`bytes`]
```
Comment (Lulalaby) [inline discord/guild.py:3676]:
```suggestion
        image: Optional[:class:`bytes`]
```

File: discord/guild.py [hunk 3]
```diff
@@ -3709,6 +3712,9 @@ async def create_scheduled_event(
         if end_time is not MISSING:
             payload["scheduled_end_time"] = end_time.isoformat()
 
+        if cover is not MISSING:
```
Comment (Lulalaby) [inline discord/guild.py:3715]:
```suggestion
        if image is not MISSING:
```

File: discord/guild.py [hunk 4]
```diff
@@ -3709,6 +3712,9 @@ async def create_scheduled_event(
...
+            payload["image"] = utils._bytes_to_base64_data(cover)
```
Comment (Lulalaby) [inline discord/guild.py:3716]:
```suggestion
            payload["image"] = utils._bytes_to_base64_data(image)
```

**Review by Lulalaby — CHANGES_REQUESTED** [review]

**Comment by Lulalaby** [comment]
It's not a breaking change

**Comment by Snawe** [comment]
> It's not a breaking change

Why is it not a breaking change? You would need to change the paramter of `edit` [here](https://docs.pycord.dev/en/stable/api/models.html#discord.ScheduledEvent.edit) from `cover` to `image`. 
`edit` does already exist. Changing a parameter in an existing function is a breaking change, isn't it?

For `create` I am with you. Since the parameter is new there, it doesn't matter.

**Comment by Lulalaby** [comment]
oh didn't knew we have it there alr

We'll target that in another pr.
But only on the next breaking release.
So don't worry about it, we add this to our todo.
You can create a issue tho for that, that would help us to keep track.

**Review by Lulalaby — APPROVED** [review]

**Comment by Snawe** [comment]
Will create an issue and attach a PR to it with the breaking changes.

**Review by BobDotCom — DISMISSED** [review]

**Review by Lulalaby — APPROVED** [review]

**Pull request merged**